packages/babel-plugin-transform-es2015-function-name/test/fixtures/function-name/await/expected.js

```diff
@@ -0,0 +1,3 @@
+export {};
+
+var obj = { await: function _await() {} };
```

Comment:
The fn.name also changes when it's derived to be `arguments` or another keyword; not much can be done about it other than omitting the name entirely and leaving it to ES6 engines to correctly derive the name...

And yes, Chrome's error is correct. You cannot use a keyword in a var/let/const binding, and explicitly named functions create a var binding with their name.